common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceMinificationServiceImpl.java

```diff
@@ -64,20 +80,45 @@
     @Value("${minify.disableOptimizations}")
     protected boolean disableOptimizations;
 
+    @Value("${minify.closure.compiler.enabled}")
+    protected boolean closureCompilerEnabled;
+
+    protected CompilerOptions compilerOptions;
+
+    protected SystemPropertiesService systemPropertiesService;
+
```

Comment:
This should be `protected Environment environment`. Our system properties are added to the Spring environment so for consistency we've been moving away from injecting the `SystemPropertiesService` in favor of using `Environment`